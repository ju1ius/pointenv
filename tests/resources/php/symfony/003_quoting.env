A="a"
B="a'b"
C='c'
D='a"b'
E="a\"b"
# newlines
F="a\nb"
G="a\rb"
H='a\nb'
I='a\rb'
J='a
b'
J="a
b"
K='c:\'
L="c:\\"