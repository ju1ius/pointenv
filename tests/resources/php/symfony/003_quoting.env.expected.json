{
    "A": "a",
    "B": "a'b",
    "C": "c",
    "D": "a\"b",
    "E": "a\"b",
    "F": "a\nb",
    "G": "a\rb",
    "H": "a\\nb",
    "I": "a\\rb",
    "J": "a\nb",
    "K": "c:\\",
    "L": "c:\\"
}